apps/angular/injection-token/src/app/data.ts

```diff
@@ -1 +1,11 @@
+import { InjectionToken } from '@angular/core';
+
 export const DEFAULT_TIMER = 1000;
+
+export const TIMER_STATE = new InjectionToken<number>('TimerState', {
+  factory: () => DEFAULT_TIMER,
+});
+
+export function getTimerProvider(useValue: number) {
```

Comment:
use provide, it's a naming convention and this way, we know you are providing something.